package.json

```diff
@@ -18,7 +18,7 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
-    "broccoli-babel-transpiler": "~5.6.1",
+    "broccoli-babel-transpiler": "~5.6.2",
```

Comment:
It is slightly odd to leave this at 5.x, but it seems totally fine to update in a follow-up...

Reply:
Fixed now.